modules/porous_flow/doc/content/modules/porous_flow/boundaries.md

```diff
@@ -94,50 +97,62 @@ environment.  The flux of fluid from the model to this environment is
+that allows for flexibility in setting combinations of pressure and temperature boundary conditions. Due to its implementation, 
+it is difficult to draw a perfect analogy to the physical flux. Nevertheless,
+[eq:fix_pp_bc] may be implemented in a number of ways, and one of the most
+common involves a [`PorousFlowPiecewiseLinearSink`](PorousFlowPiecewiseLinearSink.md)
+that follows the format of [eq:s_f_g] using $f(x,t)=C$ and $g(P-P_{\mathrm{e}})$ as a
+piecewise linear function between ordered pairs of `pt_vals` (on the x-axis) and 
+`multiplier` (on the y-axis). The function $g$ is shown in the figure below. It accepts $P-P_{\mathrm{e}}$ as an input and returns 
+a value that ends up multiplying $C$ to give a flux. $C$ can be thought of as the conductance and is specified with `flux_function = C`. $P_{\mathrm{e}}$ can be specified using an AuxVariable or set to a constant value using `PT_shift = Pe`.
+
+
+!media piecewiselinear_g_function.png style=width:50%;margin-left:10px caption=Depiction of $g(P-P_{\mathrm{e}})$ for PorousFlowPiecewiseLinearSink. The function accepts $P-P_{\mathrm{e}}$ as an input (i.e. the difference between a specified environment pressure and the pressure on the boundary element) and returns a value that multiplies $C$ to give the flux out of the domain. id=PiecewiseLinear_g_Function
+
+To set a Dirichlet boundary condition $P = P_{\mathrm{e}}$, either $C$ or the slope of $g$ should be very large. 
+It is usually convenient to make the slope of $g$ equal to 1 by setting `pt_vals = '-1E9 1E9'` and `multipliers = '-1E9 1E9'`, and then $C$ can be selected appropriately.
+Assigning a large range for `pt_vals` ensures that $g$ is defined for the porepressures encountered in the simulation, and defining $g$ for $P-P_{\mathrm{e}}>0$ and $P-P_{\mathrm{e}}<0$ allows for outflow and inflow.
+If $P - P_{\mathrm{e}}$ falls outside of the range defined in `pt_vals`, then $g = 0$ by default. This can be useful to set a boundary condition that will only allow for outflow (e.g. by using  `pt_vals = '0 1E9'`, `multipliers = '0 1E9'`).`
+
+
+The numerical value of the conductance, $C$, is
 $\rho k_{nn}k_{\mathrm{r}}/\mu/L$, must be set at an appropriate value
 for the model.
-
 Alternately, a
 [`PorousFlowPiecewiseLinearSink`](PorousFlowPiecewiseLinearSink.md)
 may be constructed with the same `pt_vals`, `multipliers` and `PT_shift`, but with `use_mobility = true` and `use_relperm =
 true`, and then $C = 1/L$.  This has three advantages: (1) the MOOSE
 input file is simpler; (2) MOOSE automatically chooses the correct
 mobility and relative permeability to use; (3) these parameters are
 appropriate to the model so it reduces the potential for difficult
-numerical situations occuring.
+numerical situations occuring. Also note, if $C \times g$ is too large, the boundary residual will be much larger than residuals within the domain. This results in poor convergance. 
```

Comment:
I suggest the "Also note..." should be a separate paragraph as it's quite important.

Reply:
Agreed.